src/components/cart/CartItem.js

```diff
@@ -41,17 +44,18 @@ export const CartItem = ({ item }) => {
 					<input
 						type="number"
 						min="1"
-						// value={stateQty}
-						// onChange={(e) => {
-						// 	setStateQty((prev) => Number(e.target.value));
-						// handleCartQty(item.id, stateQty);
-						// }}
+						value={stateQty}
+						onChange={(e) => {
+							setStateQty(Number(e.target.value));
+							console.log(stateQty);
+							handleCartQty(item.id, stateQty);
+						}}

```

Comment:
Should move to separate function

Reply:
Actually, you don't need to pass item.id and stateQty to your function. You should take it from props and state. 
Let's suppose, if it were a list then it could be done like that. everything would depend on the value of the element.
So, let's just leave point like: 
```js
onChange={handleQtyChange}
```